Octokit.fsx

```diff
@@ -0,0 +1,80 @@
+#I __SOURCE_DIRECTORY__
+#r "./Octokit/bin/Release/Net45/Octokit.dll"
+
+open Octokit
+open System
+open System.IO
+
+type Draft =
+    { Client : GitHubClient
+      Owner : string
+      Project : string
+      DraftRelease : Release }
+
+let private isRunningOnMono = System.Type.GetType ("Mono.Runtime") <> null
+
+let rec private retry count asyncF =
+    // This retry logic causes an exception on Mono:
```

Comment:
This seems to be working on Mono `master` - is there a specific build we should be using here to ensure this works on Mono, or is it too early to rely on this fix being present?